mobx/lib/add_todo_page.dart

```diff
@@ -3,8 +3,6 @@ import 'package:flutter_mobx/flutter_mobx.dart';
 import 'package:mobx_sample/model/todo.dart';
 
 class AddTodoPage extends StatefulWidget {
```

Comment:
Why not StatelessWidget ?

Reply:
I didn't change anything from this file but I will make it stateless nevertheless.

Reply:
I think it would be good to do that. It clearly then shows the power of MobX as being the complete State Manager without the need for any StatefulWidget for 99% of the use cases :-)

Reply:
Yea sure definitely I get that :)

Reply:
Hi @pavanpodila I just had a look at the Widget, the reason that thsi widget is Stateful is because its handling the validation of its form like if the user has entered the title enable the FAB and make it blue otherwise grey and disabled.

Reply:
ah ok